ods_ci/tests/Tests/200__monitor_and_manage/200__metrics/205_bias_metrics.robot

```diff
@@ -0,0 +1,199 @@
+    Log To Console    ${output}
+    Should Be Equal As Integers    ${return_code}     0   msg=Error while applying the provided file
+
+Configure User Workload Monitoring
+    [Documentation]    Configure the retention period in User Workload Monitoring for the cluster.
+    ...                This period can be configured for the component as and when needed.
+    ${return_code}    ${output}    Run And Return Rc And Output   oc apply -f ${UWM_CONFIG_FILEPATH}
+    Log To Console    ${output}
+    Should Be Equal As Integers    ${return_code}     0   msg=Error while applying the provided file
+
+Verify User Workload Monitoring Configuration
+    [Documentation]    Verifies that the ALL monitoring components for user-defined-projects
+    ...    are READY in the ${DEFAULT_MONITORING_NS} namespace
+    [Tags]    Smoke
+    ...       Tier1
+    ...       OpenDataHub
+    Wait For Pods Status  namespace=${DEFAULT_MONITORING_NS}  timeout=60
+    Log  Verified Applications NS: ${DEFAULT_MONITORING_NS}  console=yes
+
+Verify trustyai-service-operator-controller-manager Deployment
+    [Documentation]    Verifies the correct deployment of the model controller in the namespace
+    [Arguments]    ${num_replicas}=1
+    ${all_ready} =    Run    oc get deployment -n ${APPLICATIONS_NAMESPACE} trustyai-service-operator-controller-manager | grep ${num_replicas}/${num_replicas} -o  # robocop:disable
+    Should Be Equal As Strings    ${all_ready}    ${num_replicas}/${num_replicas}
+
+Install And Verify TrustyAI Service
+    [Documentation]    Install TrustyAI service CRD and verify that TrustyAI resources have spun up
+    Deploy Custom Resource    kind=TrustyAIService    namespace=${PRJ_TITLE}
+    ...    filepath=${TRUSTYAI_CR_FILEPATH}
+    Sleep   15s
+    ${return_code}    ${output}    Run And Return Rc And Output   oc get route trustyai-service -n ${PRJ_TITLE} --template={{.spec.host}}
+    Should Be Equal As Integers    ${return_code}	 0
+    Set Suite Variable    ${TRUSTY_ROUTE}    https://${output}
+
+Delete TrustyAI Service
+    [Documentation]    Delete TrustyAI service CRD
+    OpenshiftLibrary.Oc Delete    kind=TrustyAIService    namespace=${PRJ_TITLE}    name=trustyai-service
+
+Verify Model Is Registered with TrustyAI Service
+    [Documentation]    Verify the deployed model is registered with TrustyAI Service by verifying MM_PAYLOAD_PROCESSORS
+    ...                value is the TrustyAI service route
+    ${MM_PAYLOAD_PROCESSORS_Expected}=    Set Variable    http://trustyai-service.${PRJ_TITLE}.svc.cluster.local/consumer/kserve/v2
+    Wait Until Keyword Succeeds  3 min  20 sec  Verify One Model Serving Pod Exists   ${PRJ_TITLE}    label_selector=modelmesh-service=modelmesh-serving
+    ${return_code}    ${podname}    Run And Return Rc And Output  oc get pods -n ${PRJ_TITLE} -o json | jq '.items[].metadata.name' | grep 'modelmesh-serving'
```

Comment:
you can try `Get Pod Name` kw

Reply:
`Get Pod Name` kw will give name of the pod but I am fetching the environment variable named MM_PAYLOAD_PROCESSORS. Could it still be used? @bdattoma